.ci/schedule-tests/src/combine-statistics.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs')
+const path = require('path')
+const AdmZip = require('adm-zip')
+
+const processTextFile = (fileContent, durations) => {
+  const lines = fileContent.split(/\n/)
+
+  for (const line of lines) {
+    const [suite, testFile, duration] = line.split(/\s+/)
+    const durationInSeconds = parseFloat(duration)
+
+    if (!isNaN(durationInSeconds)) {
+      const key = `${suite}:${testFile}`
+      durations[key] = [...(durations[key] || []), durationInSeconds]
+    }
+  }
+}
+
+const calculateMedian = (arr) => {
```

Comment:
Javascript has come a long way in terms of what it can do out of the box, in a functional style.  Given that, I don't want to pull in a large, opinionated library that strives to solve a lot of distinct problems, including median calculation.  If we had more statistical calculations to do, we should pull in a statistical library.  As we only need `median`, having its implementation right here is OK and it gives us the ability to inspect it to verify that it does what we require.

An empty table is not something that we're going to see with our data set.